internal/provider/provider.go

```diff
@@ -227,6 +227,54 @@ func New() *schema.Provider {
 					},
 				},
 			},
+			"coder_devurl": {
+				Description: "Use this resource to define shortcuts to access applications in a workspace.",
+				CreateContext: func(c context.Context, resourceData *schema.ResourceData, i interface{}) diag.Diagnostics {
+					resourceData.SetId(uuid.NewString())
+					return nil
+				},
+				ReadContext: func(c context.Context, resourceData *schema.ResourceData, i interface{}) diag.Diagnostics {
+					return nil
+				},
+				DeleteContext: func(ctx context.Context, rd *schema.ResourceData, i interface{}) diag.Diagnostics {
+					return nil
+				},
+				Schema: map[string]*schema.Schema{
+					"agent_id": {
+						Type:        schema.TypeString,
+						Description: `The "id" property of a "coder_agent" resource to associate with.`,
+						ForceNew:    true,
+						Required:    true,
+					},
+					"name": {
+						Type:        schema.TypeString,
+						Description: "A display name to identify the DevURL.",
+						ForceNew:    true,
+						Optional:    true,
+					},
+					"icon": {
+						Type:        schema.TypeString,
+						Description: "An icon to display in the dashboard.",
+						ForceNew:    true,
+						Optional:    true,
+					},
+					"command": {
+						Type: schema.TypeString,
+						Description: "A command to run in a terminal opening this DevURL. In the web, " +
+							"this will open in a new tab. In the CLI, this will SSH and execute the command.",
+						ConflictsWith: []string{"target"},
+						Optional:      true,
+						ForceNew:      true,
+					},
+					"target": {
+						Type:          schema.TypeString,
+						Description:   "A URL to be proxied to from inside the workspace.",
+						ForceNew:      true,
+						Optional:      true,
+						ConflictsWith: []string{"command"},
```

Comment:
We don't seem to document the `ConflictsWith` requirements at all.

Reply:
True. I shall fix!